src/RpcServer/RpcServer.Blockchain.cs

```diff
@@ -0,0 +1,204 @@
+#pragma warning disable IDE0051
+#pragma warning disable IDE0060
+
+using Neo.IO;
+using Neo.IO.Json;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+using Neo.VM;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Neo.Plugins
+{
+    partial class RpcServer
+    {
+        [RpcMethod]
+        private JObject GetBestBlockHash(JArray _params)
```

Comment:
Isn't this going to change all RPC endpoint names? For me it is ok to change, just to be sure.

Reply:
https://github.com/neo-project/neo-modules/blob/ae5cdee38cc02ec062ba0d5536ea9655497a5e77/src/RpcServer/RpcServer.cs#L238

It won't change the name.